src/App.css

```diff
@@ -12,12 +12,11 @@
 
 .App .header {
   display: grid;
-  width: 1194px;
```

Comment:
So, removing this was what broke the return button. To fix this, add `width: 390px;` to the leftCluster class.